docs/synapse/userguides/storm_ref_filter.rstorm

```diff
@@ -9,286 +12,421 @@ Storm Reference - Filtering
+**relative** property name (i.e., the property name alone, including its separator character, such as
+``:ipv4``).
+
+When filtering by universal property value, only the relative property name is required.
+
+Using the relative property name allows for simplified syntax and more efficient data entry ("less
+typing"). Full property names can be used for clarity (i.e., specifying **exactly** what you want to
+filter on).
 
-- The comparator (comparison operator) specifies how *<form>* or *<prop>* is evaluated with respect
-  to *<valu>* or *<pval>*. The most common comparator is equals (``=``), although other comparators
-  are available (see below).
-- When filtering nodes by secondary property value where the value is a time (date / time), you do not
-  need to use full ``YYYY/MM/DD hh:mm:ss.mmm`` syntax. Synapse allows you to use either lower resolution
+Full property names may be **required** in cases where multiple nodes in the inbound working set have
+the same relative property name (e.g., ``inet:dns:a:ipv4`` and ``inet:url:ipv4``, as well as universal
+properties) and you only wish to filter based on the property of one of the forms.
+
+Each example below is shown using both the full property name (*<form>:<prop>*) and the relative
+property name (*:<prop>*) where applicable.
+
+.. TIP::
+  
+  When filtering nodes by a property value where the value is a time (date / time), you do not need to
+  use full ``YYYY/MM/DD hh:mm:ss.mmm`` syntax. Synapse allows you to use either lower resolution
   values (e.g., ``YYYY/MM/DD``) or wildcard values (e.g., ``YYYY/MM*``). In particular, wildcard syntax
   can be used to specify any values that match the wildcard expression. See the type-specific documentation
   for :ref:`type-time` types for a detailed discussion of these behaviors.
 
-Filters Using Standard Comparison Operators
--------------------------------------------
 
-Filter operations can be performed using any of the standard mathematical / logical comparison operators
-(comparators):
+.. _filter-prop-std-primary:
 
-- ``=``: equals (described above)
-- ``!=`` : not equals
-- ``<`` : less than
-- ``>`` : greater than
-- ``<=`` : less than or equal to
-- ``>=`` : greater than or equal to
+Filter by Primary Property Value
+++++++++++++++++++++++++++++++++
 
 **Syntax:**
 
-*<query>* **+** | **-** *<form>* | *<prop>* *<comparator>* *<valu>* | *<pval>*
+*<query>* **+** | **-** *<form>* *<operator>* *<valu>*
 
-**Examples:**
 
-*Filter by Not Equals:*
+*Filter the current working set to exclude the loopback IPv4 address (127.0.0.1):*
 
-- Filter results to exclude the domain ``google.com``:
+.. storm-pre:: [ inet:ipv4=127.0.0.1 inet:ipv4=8.8.8.8 ] -inet:ipv4 = 127.0.0.1
+::
+  
+  <query> -inet:ipv4 = 127.0.0.1
 
-.. storm-pre:: inet:fqdn=woot.com inet:fqdn=vertex.link inet:fqdn=google.com +inet:fqdn != google.com
+.. storm-pre:: [ inet:ipv4=127.0.0.1 inet:ipv4=8.8.8.8 ] +inet:ipv4 != 127.0.0.1
```

Comment:
Looks like this is missing the description that this is including only loopback rather than excluding loopback?